app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
You don’t need to inherit from ABC as your class does not realise any abstract methods